.cloud/.azure/test/test_aks_compute.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "aks-intTest",
+    "compute_type": "akscluster",
+    "agent_count": 3,
+    "vm_size": "Standard_D3_v2",
+    "location": "<your-location>",
+    "service_cidr": "<your-service-cidr>",
+    "dns_service_ip": "<your-dns-service-ip>",
+    "docker_bridge_cidr": "<your-docker-bridge-cidr>",
+    "cluster_purpose": "<your-cluster-purpose>",
+    "vnet_resource_group_name": "<your-vnet-resource-group-name>",
```

Comment:
We can probably remove unused fields. Thoughts @marvinbuss

Reply:
Yes I agree.